instrumentation/instalambda/triggers.go

```diff
@@ -5,11 +5,26 @@ package instalambda
 
 import (
 	"encoding/json"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/lambdacontext"
 )
 
 type triggerEventType uint8
 
 const (
+	// FieldT is the trace ID attribute key in a custom client context
+	fieldT = "X-INSTANA-T"
+	// FieldS is the span ID attribute key in a custom client context
+	fieldS = "X-INSTANA-S"
+	// FieldL is the trace level attribute key in a custom client context
+	fieldL = "X-INSTANA-L"
+
+	// traceParentHeader is the W3C trace parent header name as defined by https://www.w3.org/TR/trace-context/
+	traceParentHeader = "TRACEPARENT"
```

Comment:
IIRChe W3C trace context [prescribes](https://www.w3.org/TR/trace-context/#header-name) these headers names to be in lower case. We also try to keep the W3C trace context implementation specifics in the `w3ctrace` package, that already has these values [defined](https://github.com/instana/go-sensor/blob/master/w3ctrace/context.go).

Reply:
Removed from here :)